frontend/src/app/features/readers/pdf-reader/pdf-reader.component.html

```
@@ -25,6 +25,17 @@
     </div>
     }
 
+    <div class="embed-header-actions">
+      <button class="embed-action-btn save-btn" [title]="'readerPdf.docViewer.saveDocument' | transloco" [attr.aria-label]="'readerPdf.docViewer.saveDocument' | transloco"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)